src/vdom.rs

```diff
@@ -370,49 +199,101 @@ impl<Ms: Clone, Mdl, ElC: View<Ms> + 'static, GMs: 'static> App<Ms, Mdl, ElC, GM
         }
     }
 
-    /// App initialization: Collect its fundamental components, setup, and perform
-    /// an initial render.
-    pub fn run(self) -> Self {
-        self.process_cmd_and_msg_queue(
-            self.cfg
-                .initial_orders
-                .replace(None)
-                .expect("initial_orders should be set in AppBuilder::finish")
-                .effects,
-        );
-        // Our initial render. Can't initialize in new due to mailbox() requiring self.
-        // "new" name is for consistency with `update` function.
-        // this section parent is a placeholder, so we can iterate over children
-        // in a way consistent with patching code.
-        let mut new = El::empty(dom_types::Tag::Section);
-        new.children = (self.cfg.view)(self.data.model.borrow().as_ref().unwrap()).els();
+    /// Bootstrap the dom with the vdom by taking over all children of the mount point and
+    /// replacing them with the vdom if requested. Will otherwise ignore the original children of
+    /// the mount point.
+    fn bootstrap_vdom(&self) -> El<Ms> {
+        let bootstrap_behavior = *self.cfg.bootstrap_behavior.borrow();
+        let mut new = El::empty(dom_types::Tag::Placeholder);
+
+        // Map the DOM's elements onto the virtual DOM if requested to takeover.
+        if bootstrap_behavior == Some(BootstrapBehavior::Takeover) {
```

Comment:
Can we rename it to `mount_type` and `MountType` so users don't have to relearn name once we change API to this one: https://github.com/David-OConnor/seed/issues/250#issuecomment-544512869?